science/2022-03-03-science-blog-5/index_de.md

```diff
@@ -0,0 +1,898 @@
+
+Die Zahl der erstmaligen Downloads war in den ersten Monaten nach der Einführung der CWA eine gute Annäherung, muss jedoch mit der Zeit als zunehmend schlechte Approximation gesehen werden, da die CWA vom Smartphone gelöscht werden kann oder das alte Smartphone einem neuen weicht, auf dem die CWA unter einem neuen Account erneut installiert wird. Wir geben weiter unten eine [Schätzung der aktiven Nutenden aus einer Heuristik](#43-aus-einer-heuristik) an, die diese sunehmende Abweichung zwischen Downloads und aktiven Nutzenden bzw. aktiven Apps/Endgeräten präzisiert.
+
+Hinzu kommt, dass die Corona-Warn-App im Lauf der Zeit um viele weitere Features erweitert wurde (siehe Tabelle 1). Ob Nutzende in der CWA vielleicht nur das Kontakttagebuch (ab Release 1.10), das Informationsangebot (ab Release 1.11), die Check-in-Funktion für Veranstaltungen (ab Release 2.0) oder die Zertifikatsverwaltung (ab Release 2.3) nutzen, wird ebenfalls nicht ermittelt.
+
+Bei der aktiven Nutzung im erweiterten Sinne wird nicht unterschieden, welche Features genutzt werden. Jemand wird demnach als aktiver Nutzender gezählt, so lange ein beliebiges Feature genutzt wird (z. B. ausschließliche Nutzung des Zertifikatsnachweises), unabhängig davon, ob die Kernfunktionen genutzt werden.
+
+Die vom [Apple-App-Store](#62-apple-app-store) und [Google-Play-Store](#61-google-play-store) ausgewiesenen Kennzahlen sind unterschiedlich definiert und werden unterschiedlich gemessen, betreffen oft nur näherungsweise denselben Sachverhalt und lassen sich daher nicht unmittelbar summieren oder vergleichen. Zudem ist besondere Vorsicht geboten: der Begriff aktive Nutzende wird dort gänzlich anders ([homonym](https://de.wikipedia.org/wiki/Homonym)) als in diesem Blog definiert und steht für etwas, das wir potentielle Nutzende nennen könnten. Weiterhin kommt neben der Differenzierung zwischen Nutzenden und Endgeräten noch das Konzept eines Nutzenden im Sinne eines Accounts vor, was die Dinge zusätzlich verkompliziert. Dort wo sie die Nutzung betreffen, können auch diese Kennzahlen jedenfalls nur die aktive Nutzung im erweiterten Sinne beschreiben. Wir werden die Zahlen der Stores daher auch separat beschreiben.
+
+<table class="table table-striped table-hover table-condensed table-responsive" style="margin-left: auto; margin-right: auto;">
+<thead>
+<tr>
+<th style="text-align:left;">
+Release
+</th>
+<th style="text-align:left;">
+In der CWA implementierte Funktionen
```

Comment:
This table comes a bit random, there is not introduction sentence like "Im Nachfolgenden werden die Features der CWA nach Version aufgezählt." or any valuable information for the reader.

Reply:
Hi Tim, 

thank you very much for your suggestion, which we are happy to adopt. An adequate formulation will follow.

Best regards
Tobias

Reply:
PR https://github.com/corona-warn-app/cwa-website/pull/2507

Reply:
Thanks @larswmh! This can be resolved.